components/docs-chef-io/content/automate/ha_troubleshooting.md

```diff
@@ -185,15 +164,86 @@ terraform taint module.bootstrap_automate.null_resource.automate_post[2]
 While trying to deploy or upgrade Chef Automate HA, if the command did not exit gracefully, the possible error looks like as shown below:
 
 ```bash
-      Exiting as execution process completed
-UnknownError: Unable to determine error cause: Command did not exit gracefully
+      Exiting as the execution process completed
```

Comment:
Is this actually the error or modified for the grammatical correction? If so, we should stick to the error the command throws

Reply:
@kalroy @dishanktiwari2501 - This seems to be corrected for grammatical error. Reverted this.